app/mailers/teacher_mailer.rb

```diff
@@ -49,11 +49,17 @@ def application_not_submitted
   end
 
   def application_received
-    view_mail(
-      GOVUK_NOTIFY_TEMPLATE_ID,
-      to: teacher.email,
-      subject: I18n.t("mailer.teacher.application_received.subject"),
-    )
+    subject =
+      if @application_form.teaching_authority_provides_written_statement
+        I18n.t("mailer.teacher.application_received.subject.without_document")
+      else
+        I18n.t(
+          "mailer.teacher.application_received.subject.awaiting_document",
+          certificate: region_certificate_name(region),
+        )
+      end
```

Comment:
I think this should actually be the other way around, if the teaching authority provides the written statement (rather than the applicant), that's when we'd be waiting for the document.